bee-message/CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.2.0 - 2022-XX-XX
```

Comment:
Does this need a proper date?

Reply:
Not really, I don't know yet if we'll release a version before the shimmer crate